### PR TITLE
Creates a Wiretap Guide

### PR DESCRIPTION
**Pull request by JeffNeff**
Live preview: https://7729d2e95e82.ngrok.io/

### CODE REVIEW

File: docs/guides/createawiretap.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Creating a Wiretap
+In this guide we will create a Wiretap to monitor the Cloudevent traffic happening within our bridge.
+
+
+## What is a Wiretap?
+A Wiretap is a powerful debugging tool/methodology that can be used to understand the flow of events through the system by subscribing to all of the events that pass through the associated `Broker`. This is accomplished by the use of a `Trigger` to route all of the events into a logging service.
+
+
+Consider the following illustration:
```
Comment (jmcx) [inline docs/guides/createawiretap.md:9]: Can we update this to use the same style as the other guides? I think Seb mentioned using Excalidraw.

**Review by jmcx — CHANGES_REQUESTED** [review]
Awesome stuff! No comments other than the diagram style.

**Comment by sebgoa** [comment]
This uses the `kind: Bridge` which is triggerflow and therefore not open source.

File: docs/guides/createawiretap.md (hunk 1 above)
Comment (JeffNeff) [reply docs/guides/createawiretap.md:9]: done :)

**Comment by JeffNeff** [comment]
> This uses the `kind: Bridge` which is triggerflow and therefore not open source.

Updated. and removed bridge refrences.

**Review by jmcx — APPROVED** [review]
Great!

File: docs/assets/yamlexamples/simple-bridge.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+apiVersion: flow.triggermesh.io/v1alpha1
+kind: Bridge
```
Comment (sebgoa) [inline docs/assets/yamlexamples/simple-bridge.yaml:2]: we should remove this and replace with a concatenated yaml...

Comment (JeffNeff) [reply docs/assets/yamlexamples/simple-bridge.yaml:2]: what does that mean? im sorry. @sebgoa

**Pull request merged**